doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -1329,7 +1405,7 @@ The issues in this section are related to the :ref:`nrf_desktop` application.
 .. note::
     nRF Desktop is also affected by the Bluetooth LE issue :ref:`NCSDK-19865 <ncsdk_19865>`.
 
-.. rst-class:: v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
```

Comment:
I think we can add `wontfix` tag here too.

Reply:
Updated. Please check.

Reply:
I think you accidentally added it to the `NCSDK-19865: GATT Robust Caching issues for bonded peers` instead of adding it here. Please fix :)

Reply:
Fixed.